README.md

```diff
@@ -8,6 +8,28 @@ Clementine is a modern music player and library organizer for Windows, Linux and
 - Buildbot: http://buildbot.clementine-player.org/grid
 - Latest developer builds: http://builds.clementine-player.org/
 
+Opening an issue
+----------------
+### Ask a new feature
```

Comment:
"Ask for"? But I'm not a native English speaker, @hatstand or @davidsansome would know better :)